app/javascript/dashboard/i18n/locale/en/inboxMgmt.json

```diff
@@ -371,6 +371,8 @@
     "DETAILS": {
       "LOADING_FB": "Authenticating you with Facebook...",
       "ERROR_FB_AUTH": "Something went wrong, Please refresh page...",
+      "ERROR_FB_UNAUTHORIZED": "You're not authorized to perform this action. ",
+      "ERROR_FB_UNAUTHORIZED_HELP": "Please ensure you have access to the Facebook page with full control. You can read more about Facebook roles <a href=\" https://www.facebook.com/help/187316341316631\">here</a>.",
```

Comment:
Can you please re-write this to Facebook specific language? Full Control seems like a generic term and the user wouldn't probably receive anything when they search for it on Google.

Reply:
@pranavrajs "Full Control" is the term they use. I tried finding a specific term, but couldn't

ref: https://www.facebook.com/business/help/582754542592549?id=418112142508425

Reply:
@pranavrajs Should I just use the word "Admin" here?

Reply:
Let us see how this goes.